app/views/shared/_user_profile_image.html.erb

```diff
@@ -0,0 +1 @@
+<%= image_tag user.profile_image_thumbnail, class: "rounded-full" %>
```

Comment:
I think we'll need a `h-full w-full` here:

![CleanShot 2024-04-29 at 17 04 29](https://github.com/maybe-finance/maybe/assets/16676157/51be00d3-489f-4f74-a233-5a76d56251c7)

Reply:
Good spot I was testing with a perfectly square image.  Added `object-cover` too to prevent the image looking warped. Also form validations for image format which were missed!